test/install/Makefile

```diff
@@ -0,0 +1,103 @@
+NAME=syft
+
+IMAGE_NAME=$(NAME)-install.sh-env
+UBUNTU_IMAGE=$(IMAGE_NAME):ubuntu-20.04
+ALPINE_IMAGE=$(IMAGE_NAME):alpine-3.6
+BUSYBOX_IMAGE=busybox:1.35
+
+ENVS=./environments
+DOCKER_RUN=docker run --rm -t -w /project/test/install -v $(shell pwd)/../../:/project
+UNIT=make unit-local
+
+# acceptance testing is running the current install.sh against the latest release. Note: this could be a problem down
+# the line if there are breaking changes made that don't align with the latest release (but will be OK with the next
+# release)
+acceptance=sh -c '../../install.sh -b /usr/local/bin  && syft version'
```

Comment:
Would it be ok to uppercase this variable so that we make it distinct from our make target on line 31?